integration-tests/test/ovmcontext.spec.ts

```diff
@@ -74,9 +74,11 @@ describe('OVM Context: Layer 2 EVM Context', () => {
       const l1BlockNumber = await OVMContextStorage.l1BlockNumbers(i)
       expect(l1BlockNumber.toNumber()).to.deep.equal(l1Block.number)
 
-      // L1 and L2 blocks will have the same timestamp.
+      // L1 and L2 blocks will have approximately the same timestamp.
```

Comment:
Can we make the assertion of "strictly greater than" in addition to "approximately equal to"?

Reply:
Which do you mean should be greater than? - an L1 block can have at highest a timestamp of `time.Now() + 15s` which could be larger than the `time.Now` that the sequencer assigns to a transaction. This only applies if we do not trail L1 by any amount - then the configurable amount that we trail L1 by impacts which is larger

Reply:
I added another test that asserts that the L1 blocknumber and timestamps are ascending, it runs after the stress tests run